src/components/Books.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Books = (props) => {
+  const {
+    book: {
+      title,
+      author,
+      id,
+      category,
+    },
+
+  } = props;
+
+  return (
+    <>
+      <li key={id} className="books-content">
+        {title}
+        {' by '}
+        {author}
+        {' - '}
+        {category}
+      </li>
+      <button type="button" id={id}>Remove</button>
+    </>
+  );
+};
```

Comment:
[OPTIONAL] Since this component is responsible for rendering the book's details, I'd suggest calling it Book rather than Books.